go-ssb-bindings/utils.go

```diff
@@ -215,3 +215,36 @@ type healReport struct {
 	Authors  []*ssb.FeedRef
 	Messages uint64
 }
+
+// TODO: probably want to add a blobsNotify-like callback instead to reduce polling
+
+//export planetaryBearerToken
+func planetaryBearerToken() *C.char {
+	lock.Lock()
+	defer lock.Unlock()
+
+	var err error
+	defer func() {
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "planetaryBearerToken error: %+v", err)
+		}
+	}()
+
+	if sbot == nil {
+		err = ErrNotInitialized
+		return nil
+	}
+
+	if servicePlug == nil {
+		err = errors.Errorf("service plugin not loaded")
+		return nil
+	}
```

Comment:
My witcher's medallion is buzzing when I scroll through here... dunno why. @cryptix Why the nil checks here?

Reply:
> Why the nil checks here?

theoretically you can call these before calling `ssbBotInit()` which results in an immediate crash because of the nil derefs..